src/content/docs/curriculum-help.mdx

```diff
@@ -384,84 +493,74 @@ const match =
 
-There are a few methods on the `__helpers` object to remove content from the camper's code.
+Available via `__helpers.getFunctionParams(codeString)`, this method extracts parameter information from a function definition string.
 
-These do NOT need to be instantiated they are static methods.
+```js
+const funcString = 'function test(a, b = "default", c) { return a + b + c; }';
+const params = __helpers.getFunctionParams(funcString);
+// params = [
+//   { name: 'a', defaultValue: undefined },
+//   { name: 'b', defaultValue: 'default' },
+//   { name: 'c', defaultValue: undefined }
+// ]
+```
 
-### Removing Comments
+#### `.escapeRegExp()`
 
-Using `__helpers.removeCssComments()`, `__helpers.removeHtmlComments()`, or `__helpers.removeJSComments()` allows you to pass the camper's code (through the `code` variable) to remove comments matching the language's comment syntax.
+Available via `__helpers.escapeRegExp(string)`, this method escapes special regular expression characters in a string.
 
-### Removing Whitespace
+```js
+__helpers.escapeRegExp('hello.world'); // "hello\\.world"
+__helpers.escapeRegExp('test[123]'); // "test\\[123\\]"
+```
 
-Using `__helpers.removeWhiteSpace()` allows you to pass the camper's code (through the `code` variable) to remove all whitespace.
+## Python Helper
 
-## permutateRegex
+### Removing Python Comments
 
-Permutates regular expressions or source strings, to create regex matching elements in any order.
+Use `__helpers.python.removeComments(code)` to remove Python comments from code strings:
 
-```javascript
-const source1 = 'a';
-const regex1 = /b/;
-const source2 = 'c';
+```js
+const codeWithComments = `
+def test():  # This is a comment
+    return True  # Another comment
+`;
 
-permutateRegex([source1, regex1, source2]).source ===
-  new RegExp(
-    /(?:a\s*\|\|\s*b\s*\|\|\s*c|b\s*\|\|\s*a\s*\|\|\s*c|c\s*\|\|\s*a\s*\|\|\s*b|a\s*\|\|\s*c\s*\|\|\s*b|b\s*\|\|\s*c\s*\|\|\s*a|c\s*\|\|\s*b\s*\|\|\s*a)/
-  ).source;
+const cleaned = __helpers.python.removeComments(codeWithComments);
+// Returns: "\ndef test():  \n    return True  \n"
 ```
 
-Inputs can have capturing groups, but both groups and backreferrences need to be named. In the resulting regex they will be renamed to avoid duplicated names, and to allow backreferrences to refer to correct group.
+### Python AST-based Helpers
 
-```javascript
-const regex = permutateRegex(['a', /(?<ref>'|"|`)b\k<ref>/], {
-  elementsSeparator: String.raw`\s*===\s*`
-});
-
-regex.source ===
-  new RegExp(
-    /(?:a\s*===\s*(?<ref_0>'|"|`)b\k<ref_0>|(?<ref_1>'|"|`)b\k<ref_1>\s*===\s*a)/
-  ).source;
-
-regex.test('a === "b"'); // true
-regex.test("'b' === a"); // true
-regex.test('a === `b`'); // true
-regex.test(`a === 'b"`); // false
-```
+The `__helpers` object includes comprehensive Python code analysis tools through the `python` property. These helpers provide powerful Abstract Syntax Tree (AST) analysis capabilities for Python challenges.
```

Comment:
Maybe the first sentence should be removed or moved right below `## Python Helper`
```suggestion
These helpers provide powerful Abstract Syntax Tree (AST) analysis capabilities for Python challenges.
```